app/helpers/bootstrap_icons_helper.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# Helper to render Bootstrap Icons
+module BootstrapIconsHelper
+  # refer https://icons.getbootstrap.com/
+  def icon(icon, options = {})
+    file = File.read("node_modules/bootstrap-icons/icons/#{icon}.svg")
+    content_tag(:span, class: options[:class].presence) do
```

Comment:
As far as I can tell `options[:class]` and `options[:class].presence` both just return nil if there's not a `:class` key so I don't think you need to call `.presence`